docs/INSTALL.adoc

```diff
@@ -0,0 +1,127 @@
+:lang: en
+:toc:
+
+= Install LinuxCNC
+
+== Basic Instructions
+
+From the repository top level directory:
+
+* *Build LinuxCNC* +
++
+----
+cd src                                            <1>
+./autogen.sh                                      <2>
+./configure [--with-realtime=uspace|/usr/rtai...] <3>
+make clean                                        <4>
+make                                              <5>
+sudo make setuid                                  <6>
+----
+<1> Change to source directory
+<2> Builds the configure script (uses `configure.in`)
+<3> Configure the package for your system +
+    Running `configure` takes awhile. While running, it prints some
+    messages telling which features it is checking for.
+<4> Remove previous build(s) artifacts
+<5> Build/compile the software package
+<6> Set ownership and mode of built files
```

Comment:
At first glance it looks as if you have to do the steps in the code block and the numbered items.
The same for section "Run LinuxCNC" and INSTALL_es as well.

```suggestion

[NOTE]
====
<1> Change to source directory
<2> Builds the configure script (uses `configure.in`)
<3> Configure the package for your system +
    Running `configure` takes awhile. While running, it prints some
    messages telling which features it is checking for.
<4> Remove previous build(s) artifacts
<5> Build/compile the software package
<6> Set ownership and mode of built files
====
```

Reply:
I don't think that callouts still work if embedded in an admonition block !?

Reply:
As agreed, leaving it as is for the moment...
I'll add a small descriptive paragraph later to avoid the dry code excerpt